app/uk/gov/hmrc/gform/restapi/RestApiController.scala

```diff
@@ -14,23 +14,36 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.gform.config
+package uk.gov.hmrc.gform.restapi
 
 import javax.inject.Inject
 
 import play.api.libs.json.Json
-import play.api.mvc.Action
-import play.api.mvc.Results._
+import play.api.mvc.{ Action, RequestHeader }
+import uk.gov.hmrc.gform.controllers.ControllersModule
 import uk.gov.hmrc.gform.gformbackend.GformBackendModule
+import uk.gov.hmrc.gform.sharedmodel.form.{ FileId, FormId }
 import uk.gov.hmrc.play.frontend.controller.FrontendController
+import uk.gov.hmrc.play.http.HeaderCarrier
 
-import scala.concurrent.ExecutionContext.global
+class RestApiController @Inject() (
+  gformBackendModule: GformBackendModule,
+  controllersModule: ControllersModule
+)
+    extends FrontendController {
+  import uk.gov.hmrc.gform.controllers.AuthenticatedRequest._
 
-class ConfigController @Inject() (gformBackendModule: GformBackendModule) extends FrontendController {
+  //TODO: use proxy actions here
+  //TODO: proxy actions must rely on WsHttp
```

Comment:
Probably better to provide more background on this comment, if some one else were to pick up this TODO I don't think they would have enough details.